sunpy/map/sources/sdo.py

```diff
@@ -42,6 +42,23 @@ class AIAMap(GenericMap):
     * `Analysis Guide <https://www.lmsal.com/sdodocs/doc/dcur/SDOD0060.zip/zip/entry/>`__
     * Instrument Paper - :cite:t:`lemen_atmospheric_2012`
     * `wavelengths and temperature response reference <https://www.lmsal.com/sdodocs/doc/dcur/SDOD0060.zip/zip/entry/figures/aia_tel_resp.png>`__
+    
+        Examples
+    --------
+    This example shows how to create an `AIAMap` from a Fido search result.
+
+    >>> import sunpy.map
+    >>> from sunpy.net import Fido, attrs as a
+    >>> import astropy.units as u
+    >>> query = Fido.search(
+    ...     a.Time('2012-09-23T12:00:00', '2012-09-23T12:01:00'),
+    ...     a.Instrument('aia'),
+    ...     a.Wavelength(171*u.angstrom)
+    ... )
```

Comment:
Question for reviewers: Do we want to use Fido or the Sample data?

Reply:
I would say sample data personally. It'd still need to be marked as online though.